pkgs/unified_analytics/test/survey_handler_test.dart

```diff
@@ -331,20 +348,35 @@ void main() {
 [
     {
         "uniqueId": "uniqueId123",
-        "url": "url123",
         "startDate": "2023-01-01T09:00:00-07:00",
         "endDate": "2023-12-31T09:00:00-07:00",
-	"description": "description123",
-	"dismissForMinutes": "10",
-	"moreInfoURL": "moreInfoUrl123",
-	"samplingRate": "1.0",
-	"conditions": [
-	    {
-	        "field": "logFileStats.recordCount",
-	        "operator": ">=",
-	        "value": 50
-      }
-	]
+        "description": "description123",
+        "dismissForMinutes": "10",
```

Comment:
Would this be better called "snoozeForMinutes", since "dismiss" seems to be used elsewhere to mean a permanent dismissal, and "snooze" is used to meant temporary?

Reply:
Ah good call, would help reduce any confusion